rules/windows/defense_evasion_disable_windows_firewall_rules_with_netsh.toml

```diff
@@ -14,6 +14,39 @@ index = ["winlogbeat-*", "logs-endpoint.events.*", "logs-windows.*"]
 language = "eql"
 license = "Elastic License v2"
 name = "Disable Windows Firewall Rules via Netsh"
+note = """## Triage and analysis
+
+### Investigating Disable Windows Firewall Rules via Netsh
+
+The Windows Defender Firewall is a native component which provides host-based, two-way network traffic filtering for a
+device, and blocks unauthorized network traffic flowing into or out of the local device.
+
+Attackers can disable firewall rules which are intended to prevent lateral movement and command and control traffic to
+enable their operations.
```

Comment:
```suggestion
Attackers can disable defensive measures (e.g. firewall rules) that are intended to prevent lateral movement and command and control traffic to enable their operations.
```